doc/rados/operations/monitoring.rst

```diff
@@ -345,33 +345,98 @@ the following::
   you are not reaching your cluster's capacity. See `Storage Capacity`_ for 
   additional details.
 
+
+**POOLS:**  
+
 The **POOLS** section of the output provides a list of pools and the notional 
 usage of each pool. The output from this section **DOES NOT** reflect replicas,
 clones or snapshots. For example, if you store an object with 1MB of data, the 
 notional usage will be 1MB, but the actual usage may be 2MB or more depending 
-on the number of replicas, clones and snapshots.
+on the number of replicas, clones and snapshots.  
+
+- **ID:** The number of the node within the pool.
+- **STORED:** actual amount of data user/Ceph has stored in a pool. This is
+  similar to the USED column in earlier versions of Ceph but the calculations
+  (for BlueStore!) are more precise (gaps are properly handled).
+
+  - **(DATA):** usage for RBD (RADOS Block Device), CephFS file data, and RGW
+    (RADOS Gateway) object data.
+  - **(OMAP):** key-value pairs. Used primarily by CephFS and RGW (RADOS
+    Gateway) for metadata storage.
+
+- **OBJECTS:** The notional number of objects stored per pool. "Notional" is
+  defined above in the paragraph immediately under "POOLS".
+- **USED:** The space allocated for a pool over all OSDs. This includes
+  replication, allocation granularity, and erasure-coding overhead. Compression
+  savings and object content gaps are also taken into account. BlueStore's
+  database is not included in this amount.
+
+  - **(DATA):** object usage for RBD (RADOS Block Device), CephFS file data, and RGW
+    (RADOS Gateway) object data.
+  - **(OMAP):** object key-value pairs. Used primarily by CephFS and RGW (RADOS
+    Gateway) for metadata storage.
 
-- **NAME:** The name of the pool.
-- **ID:** The pool ID.
-- **USED:** The notional amount of data stored in kilobytes, unless the number 
-  appends **M** for megabytes or **G** for gigabytes.
 - **%USED:** The notional percentage of storage used per pool.
 - **MAX AVAIL:** An estimate of the notional amount of data that can be written
   to this pool.
-- **OBJECTS:** The notional number of objects stored per pool.
+- **QUOTA OBJECTS:** The number of quota objects.
+- **QUOTA BYTES:** The number of bytes in the quota objects.
+- **DIRTY:** "DIRTY" is meaningful only when cache tiering is in use. If cache
+  tiering is in use, the "DIRTY" column lists the number of objects in the
+  cache pool that have been written to the cache pool but have not flushed yet
+  to the base pool.
+- **USED COMPR:** amount of space allocated for compressed data (i.e. this
+  includes comrpessed data plus all the allocation, replication and erasure
+  coding overhead).
+- **UNDER COMPR:** amount of data passed through compression (summed over all
+  replicas) and beneficial enough to be stored in a compressed form. Here is
+  another 'ceph df detail' sample output where 500K object was compressed
+  (partially, due to unaligned object size - 448K only). So 384K at USED COMPR
```

Comment:
the descriptions look good, this example doesn't match the table though. you could drop the example compression output for now, and address in the future if you want

Reply:
https://tracker.ceph.com/issues/48156